docs/001_develop/03_client-capabilities/008_interaction/criteria-segmented-control.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: 'Criteria Segmented Control'
+sidebar_label: 'Criteria Segmented Control'
+id: client-interaction-criteria-segmented-control
+keywords: [interaction, interactive, criteria, segmented control, form]
+tags:
+- interaction
+- interactive
+- criteria
+- segmented
+- control
+- form
+sidebar_position: 8
+---
+
+import CriteriaSegmentedControlDemo from '../../../../examples/ui/client-capabilities/interaction/criteria-segmented-control.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Criteria Segmented Control
+
+The segmented control generates criteria based on the selected item.
+This criteria can be used to filter data from a Genesis dataserver endpoint.
+
+Use cases:
+* Visually and structurally group related interactive criteria.
+
+## Examples
+
+<CriteriaSegmentedControlDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+     <criteria-segmented-control></criteria-segmented-control>
+    ```
+
+    Usage
```

Comment:
Nitpick but maybe add an import statement for both `CriteriaSegmentedControlOption` and `Serialisers`. Same for the react and angular examples